src/Main.java

```diff
@@ -1,8 +1,45 @@
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
 
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
+
+        //object: gets the output of Translator class in a variable (can name this to everything, like banaan)
+        //takes the input of alp and num
+
+        Translator banaan = new Translator(alphabetic, numeric);
```

Comment:
Ik snap dat dit niet serieus is, maar probeer je zelf aan te leren echt goede variable namen te bedenken. translator zou in dit geval een prima variable naam zijn